lnrpc/routerrpc/router_backend.go

```diff
@@ -1124,10 +1118,7 @@ func (r *RouterBackend) extractIntentFromSendRequest(
 		payIntent.Amount = reqAmt
 
 		// Parse destination feature bits.
-		features, err := UnmarshalFeatures(rpcPayReq.DestFeatures)
-		if err != nil {
-			return nil, err
-		}
+		features := UnmarshalFeatures(rpcPayReq.DestFeatures)
```

Comment:
The sender is setting the field, so I think we should error out in this case? `payReq.Features` is what's extracted from the invoice at L1067, and it was not validated I think.

Reply:
see comment below

Reply:
> and it was not validated I think.

i think this is the point though right? ie, not to be too strict with feature vectors we are parsing. As long as we then read it as "required", things are fine

Reply:
> As long as we then read it as "required", things are fine

I think so, tho not sure it was skipped intentionally or accidentally, but yeah what i meant was the features parsed from an outside source has already been skipped so no need to worry about that.